### PR TITLE
Update star data

### PR DESCRIPTION
**Pull request by SevenSpheres**
Update of temperatures and bolometric corrections for all main-sequence stars and brown dwarfs. Also updated temperature for the Sun.

Might as well update *everything*. Not done yet. Source used is [Mamajek's table](https://www.pas.rochester.edu/~emamajek/EEM_dwarf_UBVIJHK_colors_Teff.txt).

### CODE REVIEW

**Comment by SevenSpheres** [comment]
Was leaving as draft because the brown dwarf bolometric corrections aren't perfect, but it should be good to merge

File: src/celengine/star.h [hunk 1]
```diff
@@ -229,8 +229,9 @@ StarDetails::getEllipsoidSemiAxes() const
 bool
 StarDetails::hasCorona() const
 {
-    // Y dwarves and T dwarves subclasses 5-9 don't have a corona
-    return spectralType[0] != 'Y' && (spectralType[0] != 'T' || spectralType[1] < '5');
+    // Don't render corona for brown dwarfs that are not hot enough to glow.
+    // See https://en.wikipedia.org/wiki/Draper_point
+    return temperature >= 798;
```
Comment (375gnu) [inline src/celengine/star.h:234]: i don't think this law can be applied here

**Review by 375gnu — CHANGES_REQUESTED** [review]

File: src/celengine/star.h (hunk 1 above)
Comment (SevenSpheres) [reply src/celengine/star.h:234]: It's better than an arbitrary cutoff at T5.

Comment (375gnu) [reply src/celengine/star.h:234]: Sun's corona has temperature of millions of K

**Review by 375gnu — CHANGES_REQUESTED** [review]

File: src/celengine/star.h (hunk 1 above)
Comment (Askaniy) [reply src/celengine/star.h:234]: > The absence of X-rays from LP 944-20 during the non flaring period is also a significant result. It sets the lowest observational limit on steady X-ray power produced by a brown dwarf star, and shows that million degree Celsius upper atmospheres, or coronas, cease to exist as the surface temperature of a brown dwarf cools below about 2500 degrees Celsius and becomes electrically neutral.

https://chandra.harvard.edu/xray_sources/browndwarf_fg.html

Comment (375gnu) [reply src/celengine/star.h:234]: @SevenSpheres do you have any updates for this pr?

Comment (SevenSpheres) [reply src/celengine/star.h]: Removed that commit, will open a separate issue later

**Review by Askaniy — APPROVED** [review]

**Review by 375gnu — APPROVED** [review]

**Review by LukeCEL — APPROVED** [review]

**Pull request merged**